common/src/main/scala/explore/implicits.scala

```diff
@@ -3,12 +3,42 @@
 
 package explore
 
+import scala.annotation.unused
+
+import cats._
 import cats.effect.ContextShift
 import cats.effect.Timer
+import cats.implicits._
 import explore.model.AppContext
 import io.chrisdavenport.log4cats.Logger
+import shapeless._
+
+trait ListImplicits {
+
+  // Adapted from https://stackoverflow.com/a/21444327/5808801
+  implicit object hnilMonoid extends Monoid[HNil] {
+    val empty = HNil
+    def combine(@unused f1: HNil, @unused f2: HNil) = HNil
+  }
+
+  implicit def hconsMonoid[H: Monoid, T <: HList: Monoid] =
```

Comment:
uh, galaxy brain level

Reply:
Yes, but not mine. Original author is attributed in a comment ;)